src/sparseml/modifiers/utils/gpt_helpers.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from abc import ABC
+
+import torch
+import torch.nn as nn
+
+
+try:
+    import transformers
+except ImportError as err:
+    transformers = None
+    transformers_err = err
+
+__all__ = []
+
+
+DEBUG = False
+_LOGGER = logging.getLogger(__name__)
+
+torch.backends.cuda.matmul.allow_tf32 = False
+torch.backends.cudnn.allow_tf32 = False
+
+
+class TerminalModuleCompressor(ABC):
```

Comment:
would be good to get rid of this since we just have 1 implementor class and don't expect any in the near future